### PR TITLE
Fix background color of reactions.

### PR DESCRIPTION
**Pull request by onurays**
WhatsApp Image 2020-10-29 at 16 26 53

WhatsApp Image 2020-10-29 at 16 28 03

WhatsApp Image 2020-10-29 at 16 28 59

### CODE REVIEW

File: vector/src/main/res/drawable/reaction_rounded_rect_shape_off.xml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+    android:shape="rectangle">
+
+    <size android:width="40dp" android:height="22dp"/>
```
Comment (bmarty) [inline vector/src/main/res/drawable/reaction_rounded_rect_shape_off.xml:5]: why we have a size here? And not in `reaction_rounded_rect_shape.xml`?

**Review by bmarty — COMMENTED** [review]
LGTM, just one question

File: vector/src/main/res/drawable/reaction_rounded_rect_shape_off.xml (hunk 1 above)
Comment (onurays) [reply vector/src/main/res/drawable/reaction_rounded_rect_shape_off.xml:5]: I have no idea. It is a copy of `rounded_rect_shape_off.xml`. I will remove and test it.

Comment (onurays) [reply vector/src/main/res/drawable/reaction_rounded_rect_shape_off.xml]: Fixed.

**Review by bmarty — APPROVED** [review]
thx!

**Pull request merged**